pom.xml

```diff
@@ -15,22 +15,46 @@
   ~ limitations under the License.
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.arpnetworking.build</groupId>
+    <artifactId>arpnetworking-parent-pom</artifactId>
+    <version>1.0.1</version>
+  </parent>
   <modelVersion>4.0.0</modelVersion>
-  <name>Yammer Extra</name>
+  <name>Metrics Client Yammer Extras</name>
   <groupId>com.arpnetworking.metrics.extras</groupId>
-  <artifactId>yammer-extra</artifactId>
-  <version>0.3.4.GRPN.2-SNAPSHOT</version>
-  <packaging>jar</packaging>
+  <artifactId>metrics-yammer-extra</artifactId>
+  <version>0.3.6-SNAPSHOT</version>
+  <packaging>pom</packaging>
   <properties>
     <main.basedir>${project.basedir}</main.basedir>
```

Comment:
This is not needed anymore. In fact with the new parent pom please revisit these to specify only what is needed. You can take a look at what I did with the metrics java client here:

https://github.com/ArpNetworking/metrics-client-java/pull/6/files